### ISSUE TITLE
Nextcloud App falling back to log in screen continuously. 

### ISSUE THREAD

**Issue by MatthiasNeuhaus** [nextcloud/android#2345]
### Actual behaviour
- As soon as app runs for some time and I browse through the files, log in screen pops up. When I log in again, few seconds later log in screen is present again.

### Expected behaviour
- I log in once and this log in details should not be forgotten.
 
### Steps to reproduce
1. Log into server.
2. Browse through files on server for 5min -> log in screen should pop up.
3. Log in again -> login screen should pop up again directly.

### Environment data
Android version: 8.0.0

Device model: Xiaomi Mi A1

Stock or customized system: Stock Android

Nextcloud app version: 3.0.3 and Dev 20180309

Nextcloud server version: 10.0.1

#### Web server error log
```
No idea where to find. This is a freenas plugin.
```

#### Nextcloud log (data/nextcloud.log)
```
No entry found.
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
cc @tobiasKaminsky since likely related to the user agent string.

@MatthiasNeuhaus just as a side note, please beware that Nc 11 will be end-of-life in May so with running 10 no matte rif this issue can/will be fixed the client will not work in the near future (which is just a guess though, since Nc10 isn't tested anymore)

**Comment by MatthiasNeuhaus** [comment]
@AndyScherzinger thanks a lot for the hint. Then I'll have to work out how to update the plugin directly out of nextcloud.

**Comment by proedie** [comment]
Hello. I got the same problem with 11.0.2 (stable).

**Comment by repsorp** [comment]
Same here

Android version: 7.1.2
Device model: Nexus5X
Stock or customized system: Stock Android
Nextcloud app version: 3.0.3 (30000399)
Nextcloud server version: 11.0.2 (stable)

**Comment by AndyScherzinger** [comment]
Hi everybody,
this bearking change in the client is due to a mandatory change in the clients user agent string which will be blocked by servers that aren't up-to-date. So if you are still running Nextcloud 11 you need to update to the latest bug fix release 11.0.8, see https://nextcloud.com/blog/nextcloud-13.0.1-12.0.6-and-11.0.8-available-with-improved-password-protection/

Other than that please consider upgrading to 12 or even better 13.0.1 since Nextcloud 11 will reach its end of life in May 2018 so there won't be any fixes anymore so you might end up with having future breaking changes within the clients.

I am closing the issue for now, please re-open if the issue persists even after upgrading to 11.0.8 or higher.
